numba/cpython/enumimpl.py

```diff
@@ -7,6 +7,7 @@
                                  lower_getattr_generic, lower_cast,
                                  lower_constant, impl_ret_untracked)
 from numba.core import types
+from numba.extending import overload_method
```

Comment:
```suggestion
from numba.core.extending import overload_method
```

Reply:
Thanks for the suggestions. I pushed an update :-)